config/tests/chime_verify_upchan.yaml

```diff
@@ -0,0 +1,330 @@
+  frame_size: num_elements * (samples_per_data_set + 64) * sizeof_float * 2
+  metadata_pool: main_pool
+  upchan_in:
+    kotekan_buffer: standard
+
+gpu_beamform_output_buffers:
+  num_frames: buffer_depth
+  frame_size: num_data_sets * (samples_per_data_set/downsample_time/downsample_freq) * num_frb_total_beams * sizeof_float
+  metadata_pool: main_pool
+  gpu_beamform_output_buffer_0:
+    kotekan_buffer: standard
+
+gain_frb_buffer:
+  num_frames: buffer_depth
+  frame_size: 2048 * 2 * sizeof_float
+  metadata_pool: main_pool
+  kotekan_buffer: standard
+
+gain_tracking_buffer:
+  num_frames: buffer_depth
+  frame_size: 2048 * 2 * num_beams * sizeof_float
+  metadata_pool: main_pool
+  kotekan_buffer: standard
+
+gpu_beamform_hfb_output_buffers:
+  num_frames: buffer_depth
+  frame_size: num_data_sets * factor_upchan * num_frb_total_beams * sizeof_float
+  metadata_pool: main_pool
+  gpu_beamform_hfb_output_buffer_0:
+    kotekan_buffer: standard
+
+cpu_output_buffers:
+  num_frames: buffer_depth
+  frame_size: num_data_sets * (samples_per_data_set/downsample_time/downsample_freq) * num_frb_total_beams * sizeof_float
+  metadata_pool: main_pool
+  cpu_output_buffer_0:
+    kotekan_buffer: standard
+
+cpu_hfb_output_buffers:
+    num_frames: buffer_depth
+    frame_size: num_data_sets * factor_upchan * num_frb_total_beams * sizeof_float
+    metadata_pool: main_pool
+    cpu_hfb_output_buffer_0:
+        kotekan_buffer: standard
+
+beamform_hfb_output_buffers:
+    num_frames: buffer_depth
+    metadata_pool: main_pool
+    beamform_hfb_output_buffer_0:
+        metadata_pool: hfb_pool
+        kotekan_buffer: hfb 
+    beamform_hfb_output_buffer_read:
+        metadata_pool: hfb_pool
+        kotekan_buffer: hfb
+
+lost_samples_buffer:
+  kotekan_buffer: standard
+  num_frames: 2 * buffer_depth
+  frame_size: samples_per_data_set * num_local_freq * num_data_sets
+  metadata_pool: main_pool
+
+compressed_lost_samples_buffer:
+    kotekan_buffer: standard
+    num_frames: 2 * buffer_depth
+    frame_size: samples_per_data_set * num_data_sets / (factor_upchan * downsample_time) * sizeof_int
+    metadata_pool: main_pool
+
+compressed_lost_samples_buffer_cpy:
+    kotekan_buffer: standard
+    num_frames: 2 * buffer_depth
+    frame_size: samples_per_data_set * num_data_sets / (factor_upchan * downsample_time) * sizeof_int
+    metadata_pool: main_pool
+
+cpu_affinity: [2,3,8,9]
+
+gen_data:
+  test_data_gen_network:
+    type: square
+    value: 153
+    kotekan_stage: testDataGen
+    out_buf: network_buffer_0
+  test_data_gen_lost_samples:
+    type: const
+    value: 0
+    kotekan_stage: testDataGen
+    out_buf: lost_samples_buffer
+  test_data_gen_upchan:
+    type: square
+    value: 153
+    kotekan_stage: testDataGen
+    out_buf: upchan_in
+
+write:
+  kotekan_stage: rawFileWrite
+  #in_buf: upchan_in
+  in_buf: network_buffer_0
+  file_name: 'upchan_in'
+  file_ext: 'dump'
+  base_dir: '.'
+
+cpu_simulate:
```

Comment:
Do we actually need to CPU simulate in this config, aren't you just dumping the results of the square wave data run through the GPU?   There isn't a comparison stage, so I think we are just doing unneeded work here?